### DISCUSSION TITLE
Crazyflie 2.0 propeller keeps flying off

### DISCUSSION THREAD

**Discussion by rowancllzz** [bitcraze/discussions#141] [Q&A]
Hi,

I recently started having problems with one of my Crazyflies in which the M4 propeller keeps popping/flying off when moving to the left. It has a flow deck attached to and I am using code which involves simple motion commands with the motion commander class on Python.

It seems to pass the power-on self test. There are no problems with the propellers when it elevates upward. However, when it moves to the left (the command I have is mc.move_distance(0.0, -0.25, 0.0, velocity=0.3), this is when the M4 propeller seems to shoot off and the Crazyflie crashes.

I'm not sure what could be the problem. I tried inserting a different motor at M4 but still get the same problems. I use time.sleep commands in my code to give the drone a chance to hover between commands. In addition, the M4 LED will flash green for a few seconds after it crashes.

Would anyone have any ideas as to what could be the problem? While I have only used this Crazyflie for about a month, I've flown it multiple times so the only guess I have is that it could be wearing out.

**Comment by knmcguire** [comment]
Hi! 

You mentioned that you replaced the motor, but have you also replaced the actual propeller as well? It could be that the hole was too big in the first place, what could happen as these propellers are injection molded due to the tolerances.

**Reply by rowancllzz** [reply]
Yes, forgot to mention but replaced the propeller as well and still no success. I also checked to make sure that it was the correct A propeller I replaced it with.

**Comment by knmcguire** [comment]
It could be that you have propellers from a time that these holes tended to be too large from production. You could put a little bit of hotglue in the propeller before mounting (careful that it does not go into the motor shaft.) Or you could contact us on contact_at_bitcraze.io for a replacement discount code for your propellers.
